checker/tests/lock/Strings.java

```diff
@@ -13,7 +13,9 @@ void StringIsGBnothing(
             @GuardSatisfied Object o3,
             @GuardedByBottom Object o4) {
         String s1 = (String) o1;
+        // :: error: (cast.incomparable)
```

Comment:
This is incompatible with the comment on line 9 - there should be no errors in this method.